orchestra/static/orchestra/project-management/data-service.es6.js

```diff
@@ -64,7 +64,7 @@ export default function dataService ($location, $rootScope, $route, orchestraApi
       dataService.ready = orchestraApi.projectInformation(this.currentProject.id)
```

Comment:
We had to update frontend part because of the following fragments:
`static/orchestra/common/js/orchestra_api.es6.js`
```
    projectInformation: function (projectId) {
      return $http.post(getApiUrl('project_information'), {
        'project_id': projectId
      })  
    },  
```

`interface_api/project_management/urls.py`
```
    url(r'^project_information/$',
        views.project_information_api,
        name='project_information'),
```

`interface_api/project_management/views.py`
```
@project_management_api_view_base
def project_information_api(request):
    """ 
    This function is used by both the project management interface
    (project admins only) and for providing project information to
    experts (only to experts associated with a project). We enforce
    both of these permissions in the view below.

    """
    project_id = load_encoded_json(request.body)['project_id']
    worker = get_object_or_404(Worker, user=request.user)
    if not (is_project_admin(request.user) or
            worker.assignments.filter(task__project=project_id).exists()):
        raise PermissionDenied
    try:
        return project_management.project_management_information(project_id)
    except Project.DoesNotExist:
        raise BadRequest('Project not found for the given id.')
```

`interface_api/project_management/project_management.py`
```
def project_management_information(project_id):
    project = Project.objects.get(id=project_id)
    project_information = get_project_information([project.id])
    project_information[project.id]['project']['status'] = dict(
        Project.STATUS_CHOICES).get(project.status, None)
...
```